INSTALL.md

```diff
@@ -57,7 +57,16 @@ helm repo update
 helm install crossplane --namespace crossplane-system crossplane-stable/crossplane
 ```
 
-## Install the DigitalOcean Crossplane Provider 
+## Install the DigitalOcean Crossplane Provider
+
+1. Find the provider installation file at [provider-digitalocean/examples/provider/install.yaml](./examples/provider/install.yaml)
+2. Run the installation:
+
+```console
+kubectl apply -f provider-digitalocean/examples/provider/install.yaml
```

Comment:
Can we update this to be the url to the raw Github content? That way you don't have to download the file.

Reply:
I've adapted the links (mine and the config one, which was already there).

As for the `install.yaml` file, it's part of this PR.